secureserver.net.plesk.json

```diff
@@ -0,0 +1,30 @@
+{
+  "providerId": "secureserver.net",
+  "providerName": "SecureServer",
+  "serviceId": "plesk",
+  "serviceName": "plesk",
+  "version": 1,
+  "description": "Enables a domain to work with Whitelabel Plesk",
+  "variableDescription": "v1 is the IP address of the server.",
+  "syncPubKeyDomain": "domain-attach.api.godaddy.com",
```

Comment:
For templates on secureserver, you want to set "shared" to true.  See the secureserver cpanel template